### PR TITLE
Some fixes for mobile mode

### PR DESCRIPTION
**Pull request by KKoukiou**
New:
Screen Shot 2021-08-12 at 11 19 04
Old:
Screen Shot 2021-08-12 at 11 14 18

### CODE REVIEW

**Comment by marusak** [comment]
I still see it like this:
Screenshot from 2021-08-12 15-03-27hmm

Are you sure you did not forget to commit something? :)

File: src/components/common/stateIcon.scss [hunk 1]
```diff
@@ -19,6 +19,8 @@
 .resource-state-text-cell {
```
Comment (marusak) [inline src/components/common/stateIcon.scss:19]: Those badges are not under `.resource-state-text-cell`

**Comment by marusak** [comment]
Oh wait, I was looking at VM details, this was on overview. But still, can we also fix it on details? :)

File: src/components/common/stateIcon.scss (hunk 1 above)
Comment (KKoukiou) [reply src/components/common/stateIcon.scss:19]: Good catch, fixed.

**Review by marusak — APPROVED** [review]
Thanks!

**Pull request merged**